pkg/test/framework/runtime/components/apps/kube.go

```diff
@@ -272,10 +268,25 @@ func appSelector(serviceName string) string {
 }
 
 type kubeComponent struct {
-	scope       lifecycle.Scope
-	deployments []*deployment.Instance
-	apps        []components.App
-	env         *kube.Environment
+	scope lifecycle.Scope
+	// requiredApps is specified by the test.
```

Comment:
This is hard to understand, maybe just delete this comment.

Reply:
Done.